src-files/def-config.yaml

```diff
@@ -56,3 +56,12 @@ verification:
   # It is an optional parameter, so it can be omitted.
   ignoreRefs:
     []
+
+# Parameters of scanners for various file types.
+scanners:
+
+  markdown:
+    # Type of repository.
+    #
+    # This affects which anchors are generated for headers.
+    repositoryType: GitHub
```

Comment:
Maybe let's call it `flavor` or `format` or something like that? There are various specifications of markdown: [CommonMark](https://commonmark.org/) is probably the most popular basic specification and GitHub-flavored markdown is an extension of it. GitLab-flavored markdown is another extension of CommonMark (it's CommonMark + some stuff AFAIK). So I think we are interested not in _repository type_ or _service where you push your repository_, but rather in markdown flavor that you follow. For example, if you use neither GitLab nor GitHub, there is still a chance that your service mandates GitHub-flavored markdown for example, so it will make sense to write `flavor: GitHub`, but less sense to write `repositoryType: GitHub`.

In future versions I guess we can also support `flavor: CommonMark` here for example.

Reply:
Hmm, sounds pretty good, gonna do.